src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -3356,22 +3380,41 @@ static void _cloneNode(SvgNode* from, SvgNode* parent, int depth)
 }
 
 
-static void _clonePostponedNodes(Array<SvgNodeIdPair>* cloneNodes, SvgNode* doc)
+static void _clonePostponedNodes(Inlist<SvgNodeIdPair>* cloneNodes, SvgNode* doc)
 {
-    ARRAY_FOREACH(p, *cloneNodes) {
-        auto nodeIdPair = *p;
-        auto defs = _getDefsNode(nodeIdPair.node);
-        auto nodeFrom = _findNodeById(defs, nodeIdPair.id);
-        if (!nodeFrom) nodeFrom = _findNodeById(doc, nodeIdPair.id);
-        if (!_findParentById(nodeIdPair.node, nodeIdPair.id, doc)) {
-            _cloneNode(nodeFrom, nodeIdPair.node, 0);
-            if (nodeFrom && nodeFrom->type == SvgNodeType::Symbol && nodeIdPair.node->type == SvgNodeType::Use) {
-                nodeIdPair.node->node.use.symbol = nodeFrom;
+    auto nodeIdPair = cloneNodes->front();
+    while (nodeIdPair) {
+        auto defs = _getDefsNode(nodeIdPair->node);
+        auto nodeFrom = _findNodeById(defs, nodeIdPair->id);
+        if (!nodeFrom) nodeFrom = _findNodeById(doc, nodeIdPair->id);
+
+        if (!_findParentById(nodeIdPair->node, nodeIdPair->id, doc)) {
```

Comment:
It seems more efficient to have the code that gets nodeFrom inside the if-condition.

```suggestion
        if (!_findParentById(nodeIdPair->node, nodeIdPair->id, doc)) {
            auto defs = _getDefsNode(nodeIdPair->node);
            auto nodeFrom = _findNodeById(defs, nodeIdPair->id);
            if (!nodeFrom) nodeFrom = _findNodeById(doc, nodeIdPair->id);
```